### PR TITLE
feat: use core-library in 0.2 eval image and RAG eval jobs

### LINKED ISSUE #849: Should we maintain the dedicated ragas_once tool, or update eval image to use core-library?
Ragas_once tool has been copied into https://github.com/kubeagi/core-library and updated to match langchain 0.1.x and ragas 0.1.x. But the eval image is still using the old code here. Should we maintain the dedicated version here, or build a new eval image to use codes in core-library?

### PR DESCRIPTION
**Pull request by Lanture1064**
#### What type of PR is this?

/kind cleanup

#### Which issue(s) this PR fixes

Fixes #849

#### Special notes for your reviewer
Also remove and deprecate dedicated ragas_once

### CODE REVIEW

File: pkg/evaluation/jobs.go [hunk 1]
```diff
@@ -40,6 +40,7 @@ const (
 	defaultPVCMountPath = "/data/evaluations"
 	defaultTestRagFile  = "ragas.csv"
 	defaultMCImage      = "kubeagi/minio-mc:RELEASE.2023-01-28T20-29-38Z"
+	defaultEvalImage    = "kubeagi/arcadia-eval:v0.2.0"
```
Comment (bjwswang) [inline pkg/evaluation/jobs.go:43]: how v0.2.0 got built?

Comment (Lanture1064) [reply pkg/evaluation/jobs.go:43]: Identical. 
https://hub.docker.com/layers/kubeagi/arcadia-eval/v0.1.0/images/sha256-dbde5aa5c25e6464f3226031ec220631cd1385c1073100ce9f658c75087c7cf1?context=explore

https://hub.docker.com/layers/kubeagi/arcadia-eval/v0.2.0/images/sha256-66c8506bf663d3b8be3c6d0e24720a69d733fc9907bc5020a2d04eea57305311?context=explore

We re-tagged a v0.2 image to v0.1 and pushed to hub 2 weeks ago due to this value setting issue. Rename this ( or considerably make it some env param) is a better option.

Comment (bjwswang) [reply pkg/evaluation/jobs.go:43]: we can keep v0.1.0 image tag unless image really get updated

**Comment by bjwswang** [comment]
@Lanture1064  Let's finish this issue https://github.com/kubeagi/core-library/issues/51 ,then deprecate ragas_once from our main repo

File: deploy/evaluation/Dockerfile [hunk 2]
```diff
@@ -32,10 +32,17 @@ ARG PYTHON_INDEX_URL=https://pypi.mirrors.ustc.edu.cn/simple/
...
-COPY ./pypi/ragas_once ./ragas_once
-WORKDIR /usr/local/bin/ragas_once
-RUN python setup.py bdist_wheel -d /usr/local/bin/ragas_once/dist
-RUN pip install dist/ragas_once-0.0.1-py3-none-any.whl
\ No newline at end of file
+# Install core-library
+RUN apt-get install -y git make gcc
+RUN git clone https://github.com/kubeagi/core-library.git
+WORKDIR /usr/local/bin/core-library
+RUN make install
```
Comment (bjwswang) [inline deploy/evaluation/Dockerfile:41]: @Lanture1064 This is cool. LGTM

Comment (Lanture1064) [reply deploy/evaluation/Dockerfile]: The eval image just need the `cli[eval]` part, so maybe we could modify the makefile?

**Review by bjwswang — APPROVED** [review]

**Comment by bjwswang** [comment]
@Lanture1064  Good work! Thanks!

**Pull request merged**